src/controller/discoverymgr/mnedc/client/client.go

```diff
@@ -442,19 +446,12 @@ func (c *Client) NotifyBroadcastServer(configPath string) error {
 		return err
 	}
 
-	file, err := os.Open(configPath)
+	serverIP, _, err := getMNEDCServerAddress(configPath)
 
 	if err != nil {
-		log.Println(logPrefix, "cant read config file from", configPath, err.Error())
+		log.Println(errors.New("Cannot read config file," + err.Error()))
```

Comment:
Do you have any reason to use error.New() ?

Reply:
No, no specific reason. Just did that to keep the flow consistent wherever I was reading YAML files. If you feel old one was better, I would do that. Please suggest !